### DISCUSSION TITLE
404 download link

### DISCUSSION THREAD

**Discussion by drag0nst0ne** [trakt-tools/universal-trakt-scrobbler#196] [Q&A]
How do I even download it? Both chrome and firefox links are offline and github releases page is empty.

**Comment by NoName95x** [comment]
Is this project dead?

**Comment by rafaelgomesxyz** [comment]
It's not released yet. We're working on a stable version for release.

**Comment by RaghavPrabhakar66** [comment]
Any ETA on release? Is their a beta version available?

**Comment by rafaelgomesxyz** [comment]
Trying to get it released by the end of the month, but we'll see. There's no beta currently, although you can manually install the extension using the development steps.

**Comment by dm3ch** [comment]
Any updates from Juny?

Do you need any help needed with testing or something else to make release happen?

**Comment by nadernassif** [comment]
I tried to build the app, tested all the steps until pm run zip but there is no "zip" in the npm run-script file. thus it fails.
Also interested here and willing to test/assist with dev

**Comment by rafaelgomesxyz** [comment]
Sorry, I've just been busy. I'll try my best to get an update for you guys by the end of the week.

**Comment by KrisDowns** [comment]
Any update on this? If not it's totally understandable with all the holiday craziness. I can't wait to try it since I'm apparently too dumb to build from the .md instructions.

**Comment by rafaelgomesxyz** [comment]
I'll be able to work on it for sure this week, since my vacation from work begins, so I'll have plenty of time. Hopefully a release soon.
